src/views/start/index.jsx

```diff
@@ -95,8 +93,33 @@ export default class Start extends Component {
 
     helpWindow.on('closed', () => {
       helpWindow = null;
-    });
-  };
+    })
+  }
+
+  onQueryKeyDown(e) {
```

Comment:
if you use arrow synatx. onQueryKeyDown=(e)=>{ you dont have to bind the function to "this"